content/en/synthetics/dashboards/api_test.md

```diff
@@ -2,23 +2,24 @@
 title: Synthetic API Test Performance Dashboard
 kind: documentation
 further_reading:
-- link: '/synthetics/ci_results_explorer'
+- link: '/continuous_testing/explorer/'
   tag: 'Documentation'
-  text: 'Learn about the CI Results Explorer'
+  text: 'Learn about the Synthetic Monitoring & Continuous Testing Explorer'
 ---
 
 ## Overview
 
 The [API test performance dashboard][1] provides insights about your entire stack and events. It shows:
 
 - **API test types**: View your network levels' average response time, latency, or lookup time, along with transaction timings and response time by location by test type.
-- **Events**: View events triggered for all of your API tests and filter for specific tests using the template variables at the top of the dashboard.
 
-Click the pink Watchdog icon to open the [**Watchdog Insights**][2] side panel and analyze outstanding anomalies in your application performance or triggered monitor alerts.
+  {{< img src="synthetics/dashboards/api_test_performance_dashboard.png" alt="Out-of-the-box Synthetics API test performance dashboard" style="width:100%" >}}
 
-{{< img src="synthetics/dashboards/api_test_performance_dashboard.png" alt="Out-of-the-box Synthetics API test performance dashboard" style="width:100%" >}}
+- **Events**: View events triggered for all of your API tests and filter for specific tests using the template variables at the top of the dashboard.
+  
+  {{< img src="synthetics/dashboards/api_test_performance_events.png" alt="Events section of the Synthetics API test performance dashboard" style="width:100%" >}}
 
-{{< img src="synthetics/dashboards/api_test_performance_events.png" alt="Events section of the Synthetics API test performance dashboard" style="width:100%" >}}
+Click the pink Watchdog icon to open the [**Watchdog Insights**][2] side panel and analyze outstanding anomalies in your application performance or triggered monitor alerts.
```

Comment:
I couldn't see this when I looked at the app, is there something you need to do to enable this?

Reply:
```suggestion
```